proposals/relaxed-simd/Overview.md

```diff
@@ -44,3 +44,78 @@ Example of some instructions we would like to add:
   and [meeting
   notes](https://github.com/WebAssembly/meetings/blob/master/main/2021/CG-03-16.md)
 - [SIMD proposal](https://github.com/WebAssembly/simd)
+
+## How to suggest instructions
+
+1. [File an issue](https://github.com/WebAssembly/relaxed-simd/issues/new)
+2. In the body of the issue, copy the template below and answer the questions (an example of FMA has been given below):
```

Comment:
GH has issue templates, though this would add files to the repo: https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

Reply:
Nice, we can probably use that, and it should be fine to add new files... Maybe when we merge it back into spec we just have to remove it.

Reply:
Move this to https://github.com/WebAssembly/relaxed-simd/pull/14, lmk what you think!